test/dynamo/test_higher_order_ops.py

```diff
@@ -692,6 +692,35 @@ def f(k):
         x = torch.randn(3, 3)
         self._test_wrap_simple(g, (x,), 2)
 
+    def test_wrap_kwarg(self):
```

Comment:
Could we add some more tests for the following:
- a kwarg is None
- a kwarg is some unsupported type (like a List of Tensor?)